www/includes/easyparliament/templates/html/search/results.php

```diff
@@ -17,104 +18,104 @@
+                            <p class="search-result__description"><?= isset($result['speaker']) ? $result['speaker']['name'] . ': ' : '' ?><?= $result['extract'] ?></p>
+                        </div>
+                      <?php } ?>
 
                     <hr>
 
-                    <?php if ( $pagination_links ) { ?>
-                    <div class="search-result-pagination">
-                        <?php if ( isset($pagination_links['prev']) ) { ?>
-                        <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
-                        <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
-                        <?php }
-                        foreach ( $pagination_links['nums'] as $link ) { ?>
-                        <a href="<?= $link['url'] ?>"<?= $link['current'] ? ' class="search-result-pagination__current-page"' : '' ?>><?= $link['page'] ?></a>
-                        <?php }
-                        if ( isset($pagination_links['next']) ) { ?>
-                        <a href="<?= $pagination_links['next']['url'] ?>" title="Next page">&gt;</a>
-                        <a href="<?= $pagination_links['lastpage']['url'] ?>" title="Final page">&gt;&gt;</a>
-                        <?php } ?>
-                    </div>
-                    <?php } ?>
-                <?php } ?>
+                      <?php if ( $pagination_links ) { ?>
+                        <div class="search-result-pagination">
+                          <?php if ( isset($pagination_links['prev']) ) { ?>
+                            <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
+                            <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
+                          <?php } ?>
+                          <?php foreach ( $pagination_links['nums'] as $link ) { ?>
+                            <a href="<?= $link['url'] ?>"<?= $link['current'] ? ' class="search-result-pagination__current-page"' : '' ?>><?= $link['page'] ?></a>
+                          <?php } ?>
+                          <?php if ( isset($pagination_links['next']) ) { ?>
+                            <a href="<?= $pagination_links['next']['url'] ?>" title="Next page">&gt;</a>
+                            <a href="<?= $pagination_links['lastpage']['url'] ?>" title="Final page">&gt;&gt;</a>
+                          <?php } ?>
+                        </div>
+                      <?php } ?>
+                  <?php } # end of !isset($error) ?>
+                </div>
+
+                <?php include 'sidebar.php' ?>
             </div>
+          <?php } ?>
 
-            <?php include 'sidebar.php' ?>
-        </div>
-        <?php } ?>
+            <?php include 'form_options.php'; ?>
```

Comment:
The sidebar can currently include a form (a "mini survey", looks like); if you ended the "if results" statement after the end of the primary div, included form_options there, then add another "if results" statement for sidebar include and closing div, would that be okay?